src/styles/typography/index.md.njk

```diff
@@ -99,6 +99,25 @@ Use the `govuk-link--no-visited-state` modifier class where it is not helpful to
 
 {{ example({group: "styles", item: "typography", example: "link-no-visited-state", html: true, open: true}) }}
 
+If it's an external link to a non-government website, make that clear in the link text. For example, 'see advice on writing link text from [name of organisation]'. There's no need to say explicitly that you're linking to an external site. [Do not use an external link icon](https://designnotes.blog.gov.uk/2016/11/28/removing-the-external-link-icon-from-gov-uk/).
```

Comment:
Could we consider adding something along the lines of 
> You should only link to websites you trust.

Reply:
I'm inclined to think this is only useful if we provide more detail on how to identify a trust-worthy website.

GOV.UK does that: https://www.gov.uk/guidance/content-design/links) - but I'm inclined to leave it out for now. What do you think?

Reply:
Yes okay let's not do anything about it for now, I don't think linking to the GOV.UK guidance makes sense.